Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -55,9 +53,47 @@ export type StatusBarAnimation = $Enum<{
   slide: string,
 }>;
 
-type DefaultProps = {
-  animated: boolean,
-};
+type StatusBarProps = $ReadOnly<{|
```

Comment:
I think I'd prefer if you just named these types `Props`. I believe that is consistent with all the other files

Reply:
Renamed it